IBM Process Mining Dashboards/README.md

```diff
@@ -0,0 +1,10 @@
+# IBM Process Mining
+API client samples
+
+## Export dashboard data
+
+The sample application will execute the Analytics dashboards marked as "exportable" and outputs their results to server-side configured S3 bucket.
```

Comment:
Can you please explain that this is valid for Process Mining 1.12.0, and explain where I can find the dashboard exported data.